CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 This format is based on [Keep A Changelog](https://keepachangelog.com/en/1.0.0).
 
+## 3.6.1 -- 2022-09-20
+
+### Added
+
+* `Aeson` instances for Plutus types (in the `Plutarch.Orphans` module)
```

Comment:
Note the `newtype` being exported from that module, as this is fairly unusual. Normally, `Orphans` modules don't export anything.

Reply:
I don't think we actually need that export...

Reply:
I'll check

Reply:
Don't need it. Its been removed.